src/pvrclient-nextpvr.cpp

```diff
@@ -789,10 +798,17 @@ PVR_ERROR cPVRClientNextPVR::GetChannels(ADDON_HANDLE handle, bool bRadio)
   std::string      stream;
   LOG_API_CALL(__FUNCTION__);
 
-  m_channelTypes.clear();
+  std::map<int, std::pair<bool, bool>>::iterator  itr = m_channelDetails.begin();
+  while (itr != m_channelDetails.end()) {
+    if ( itr->second.second == (bRadio == true) )
```

Comment:
Spaces not needed at start/end of round brackets: `if (itr->second.second == (bRadio == true))`

Reply:
OK

Reply:
OK